apps/studio/pages/project/[ref]/branches/merge-requests.tsx

```diff
@@ -323,77 +326,84 @@ const MergeRequestsPage: NextPageWithLayout = () => {
   )
 }
 
-MergeRequestsPage.getLayout = (page) => {
-  const MergeRequestsPageWrapper = () => {
-    const router = useRouter()
-    const { ref } = useParams()
-    const project = useSelectedProject()
-    const gitlessBranching = useFlag('gitlessBranching')
-    const isBranch = project?.parent_project_ref !== undefined
-    const projectRef =
-      project !== undefined ? (isBranch ? project.parent_project_ref : ref) : undefined
-    const { data: branches } = useBranchesQuery({ projectRef })
-    const previewBranches = (branches || []).filter((b) => !b.is_default)
-    const { mutate: updateBranch, isLoading: isUpdating } = useBranchUpdateMutation({
-      onError: () => {
-        toast.error(`Failed to update the branch`)
-      },
-    })
-    const handleMarkBranchForReview = (branch: Branch) => {
-      if (branch.id && projectRef) {
-        updateBranch(
-          {
-            id: branch.id,
-            projectRef,
-            requestReview: true,
+const MergeRequestsPageWrapper = ({ children }: PropsWithChildren<{}>) => {
```

Comment:
just shifting this outside - ideally we don't declare components within components